### PR TITLE
Cache PHP-CS-Fixer always

### PR DESCRIPTION
**Pull request by GawainLynch**

### CODE REVIEW

File: bin/cs [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+CACHE=".php_cs_cache"
```
Comment (ro0NL) [inline bin/cs]: `.var`? for better reuseability

File: bin/cs [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+if [[ ! -d ${CACHE} ]] ; then
```
Comment (ro0NL) [inline bin/cs]: i believe, at this point, im quoting everything :P `-d "${..}"` and `mkdir "${...}"`.

Comment (GawainLynch) [reply bin/cs]: Erm … Nope, they're not. I've updated the ones in this file though.

File: bin/cs (hunk 1 above)
Comment (GawainLynch) [reply bin/cs]: What about just `%root%/var/`? (no dot)

Comment (ro0NL) [reply bin/cs]: :+1:

**Comment by GawainLynch** [comment]
Down to 0.39 seconds for the CS now :wink:

File: .php_cs.dist [hunk 3]
```diff
@@ -15,9 +16,14 @@ $rules = [
...
+}
+
 return Config::create()
-    ->setRules($rules)
-    ->setFinder(Finder::create()->in(__DIR__))
     ->setUsingCache(true)
+    ->setCacheFile($cacheDir . '/php_cs.cache')
+    ->setRules($rules)
     ->setRiskyAllowed(true)
+    ->setFinder(Finder::create()->in(__DIR__))
```
Comment (ro0NL) [inline .php_cs.dist]: shouldnt this be `__DIR__.'/src'` btw? to avoid unnecessary dir traversal

**Pull request merged**